.ci/targets/macos_build_all_packages.yaml

```diff
@@ -1,8 +1,10 @@
 tasks:
   - name: prepare tool
     script: .ci/scripts/prepare_tool.sh
+    infra_step: true # Note infra steps failing prevents "always" from running.
   - name: create all_packages app
     script: .ci/scripts/create_all_packages_app.sh
+    infra_step: true # Note infra steps failing prevents "always" from running.
```

Comment:
Does this step (same as in other yaml files) belong to infra?

Reply:
Yes assuming you are asking about if this step is infrastructure around getting a test to start running. This pr is not trying to transfer any additional responsibility to the infra-team. What is important here is in our tooling if prepare_tool or create all_packages app fails. It is not a test failure but an infra failure. Also it is likely to happen to every test that uses the same script at the same time.

Reply:
We'll have to see in practice; this could also potentially fail in the roller if the `flutter create` output changed, which wouldn't really be infra.

I would lean slightly toward not marking it, but we can always adjust later if it's an issue.